src/components/molecules/UserInfo/UserInfo.jsx

```diff
@@ -19,7 +19,7 @@ export const UserInfo = ({ userID, onClick, onPost }) => {
         const { profile_picture_url, displayName } = userData.data();
 
         return (
-            <div className={onPost ? '' : 'userinfo'}>
+            <div className={onPost ? '' : `user__info ${customClass}`}>
                 <ImageComponent
                     className='atom__image'
```

Comment:
I think that you changed the class name for image in SCSS, but forgot to change it also in the JSX component